### DISCUSSION TITLE
Integrate in Google calendar instead of local Home Assistant calendar

### DISCUSSION THREAD

**Discussion by AndiK47** [mampfes/hacs_waste_collection_schedule#1612] [Q&A]
Hi,

thanks for this awesome calendar feature!
I just have a question, since I use google calendar on most of my devices, I would love to integrate also this waste collection schedule in there. I have them both in my Home Assistant calendar, but I don´t know if there is a way to get the calendar items to the google calendar.

Do you know of a method to do that?

**Comment by 5ila5** [comment]
basically same request as in #1479. We use the Home-assistant calendar entity (<https://www.home-assistant.io/integrations/calendar>) to display events. I could not find an easy solution to export or synchronize theses calendar entities.
We may be able to a output_file argument to output all events as ICS files which could be imported into other calendars but this is currently not planed/worked on.
